tests/test_commands.py

```diff
@@ -25,6 +25,7 @@
 
 class InlineTaskApiService(TaskApiService):
     def __init__(self, work_dir: Path):
+        asyncio.get_child_watcher()
```

Comment:
What does it do? Is it needed here?

Reply:
this is to enable running multiple threads with asyncio. I got an error when i changed `exec_cmd()` to `async exec_cmd()`

Reply:
Interesting. Please put a comment here, since this looks like no-op getter.

Reply:
done, please resolve this comment when you like the addition